src/resources/i18n/translations/en.json

```diff
@@ -1,378 +1,378 @@
 {
-	"18 per year after that": "$18 per year after that",
+	"18 per year after that": "$18 yearly afterwards",
 	"A-Z": "A-Z",
-	"ARIA HINT - Contact button": "open your email client and email us.",
+	"ARIA HINT - Contact button": "open your e-mail program and send us an e-mail.",
 	"ARIA HINT - Current playback time": "Current playback time",
 	"ARIA HINT - Currently selected filter": "Currently selected filter",
 	"ARIA HINT - Error something went wrong with playing this item": "Could not play this.",
 	"ARIA HINT - Hide podcast settings": "Hide podcast settings",
-	"ARIA HINT - Max playback speed": "Set the highest available playback speed for your player",
-	"ARIA HINT - Membership - Create and share clips": "Create and share clips in Podverse premium.",
-	"ARIA HINT - Membership - Create playlists": "Create playlists in Podverse premium.",
-	"ARIA HINT - Membership - Download a backup of your data": "Download a backup of your data in Podverse premium.",
+	"ARIA HINT - Max playback speed": "Set max playback speed for your player",
+	"ARIA HINT - Membership - Create and share clips": "Create and share clips with Podverse premium.",
```

Comment:
Everything is now "Podverse premium" here. No "Podverse Premium", no "subscription" etc.